### ISSUE TITLE
Firefox upgrade nag message

### ISSUE THREAD

**Issue by duckness** [Spittie/4chan-x#24]
Some of us use the ESR releases of Firefox. It would be nice to have an option to disable the nag messages (or have it show up just once)

**Comment by Spittie** [comment]
Everything works fine? I can lower (maybe disable? I'll think about it) the nag, with the note that I officially support only the latest stable version of Firefox and Chrome.

**Comment by duckness** [comment]
Yes everything does work fine, it's kind of retarded to see the message pop up whenever I refresh/open a new tab/etc though. IMO, having the message pop up once and only once (or having an option to click "do not show this again") is preferable.

**Comment by zixaphir** [comment]
The ESR warning was from Mayhem, and though I support the sentiments of the message, seaweed fixed the only bug that existed in 4chan X on ESR awhile back.
